### PR TITLE
[Native/TFLite] average latency printer

### PR DESCRIPTION
**Pull request by helloahn**
this example prints the average latency of a pipeline consisting of TFLite & Mediapipe

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

see also: https://github.com/nnstreamer/nnstreamer/issues/2345

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+{
+  const gchar path[] = "./mediapipe/models";
+
+  gboolean failed = FALSE;
+  gchar *model = g_build_filename (path, "ssdlite_object_detection.tflite", NULL);
+
+  /* check the model files */
+  if (!g_file_test (model, G_FILE_TEST_EXISTS)) {
+    g_critical ("Failed to get model files.");
+    g_print ("Please enter as below to download and locate the model.\n");
```
Comment (gichan-jang) [inline native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc:147]: The guide to download the model is omitted.

File: native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+  /* init app variable */
+  app = g_new0 (app_data_s, 1);
+  g_assert (app);
+
+  /* load model files */
+  g_assert (load_model_files (app));
+
+  /* init pipeline */
+  pipeline =
+  /* framework=mediapipe */
```
Comment (gichan-jang) [inline native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc:211]: How about controlling with argument?

Comment (wooksong) [reply native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc:211]: Sounds great!

File: native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc (hunk 1 above)
Comment (helloahn) [reply native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc]: it's added

File: native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc (hunk 2 above)
Comment (helloahn) [reply native/example_object_detection_tensorflow_lite_appsrc/nnstreamer_example_object_detection_tflite_appsrc.cc]: it's added

**Comment by leemgs** [comment]
LGTM

**Pull request merged**